lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -282,6 +288,20 @@ namespace :foreman_tasks do
       end
     end
 
+    def with_error_handling(task, what = 'task')
+      yield
+    rescue StandardError => e
+      resolution = SKIP_ERRORS ? ', skipping' : ''
+      puts _("WARNING: #{what} failed to export#{resolution}. Additional task details below.")
```

Comment:
This translation won't work.
Exported string do not match the string that will actually be translated.
Should we interpolate the string instead?